### DISCUSSION TITLE
Markdown embed diagram perspective

### DISCUSSION THREAD

**Discussion by jbindinga** [structurizr/lite#60] [Q&A] [answered]
### Question

Hi all,

I’m using structurizr lite to generate documentation for my project with the documentation markdown feature. In my diagrams and model I’m using perspectives to highlight specific contexts. Now I want to embed diagrams in my markdown files with a specific perspective, but I cannot find any way to do so.

Is there an option to embed a diagram in markdown with a perspective selected?

Thanks!
Jeffrey

**Comment by simonbrowndotje** [comment, marked as answer]
Yes, see https://structurizr.com/help/documentation/diagrams for details.

**Reply by jbindinga** [reply]
Oh I missed that somehow, taht’s exactly what I was looking for. Works like a charm.

Thanks for the quick reply!!
Jeffrey
